substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1948,6 +1948,20 @@ impl pallet_statement::Config for Runtime {
 	type MaxAllowedBytes = MaxAllowedBytes;
 }
 
+parameter_types! {
+	pub MbmServiceWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+}
+
+impl pallet_migrations::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Migrations = ();
+	type CursorMaxLen = ConstU32<65_536>;
+	type IdentifierMaxLen = ConstU32<256>;
```

Comment:
Pretty random honestly.  
The only considerations on the upper bound are the PoV since only one of these will be accessed per block. It should therefore be okay to use something like 4MiB.  
Same for the Identifier. For the lower bound, we only assume that the cursor can hold a random key of a map and these map keys are normally not that large. If needed, it can be increased at any time by the runtime dev.

Reply:
There is now a check in `integrity_test` that should prevent most misconfigurations.